packages/markdown/components/Anchor.jsx

```diff
@@ -17,9 +17,13 @@ function getHref(href, baseUrl) {
     return `${base}/reference-link/${ref[1]}`;
   }
 
+  // we need to perform two matches for changelogs in case
+  // of legacy links that use 'blog' instead of 'changelog'
   const blog = href.match(/^blog:([-_a-zA-Z0-9#]*)$/);
-  if (blog) {
-    return `${base}/blog/${blog[1]}`;
+  const changelog = href.match(/^changelog:([-_a-zA-Z0-9#]*)$/);
```

Comment:
@domharrington @erunion can one of you spot check my regex here? It doesn't appear to correctly find this snippet in the [test case](https://github.com/readmeio/api-explorer/pull/223/files#diff-b0e081d28e277b65b24526d9e22d899dR63) and I have no idea why

Reply:
Fixed: https://github.com/readmeio/api-explorer/pull/223/commits/49626ad3bf0ac93a320f379143a49d9b3256aea4